src/database/utils/settings/model_constants/user_media_uploads.py

```diff
@@ -0,0 +1,7 @@
+
+
+from database.utils.settings.model_constants.events import EventConstants
+
+
+class UserMediaConstants(EventConstants):
+    pass
```

Comment:
Is this completed?

Reply:
Yes @BradHN1, I currently need the exact methods and properties that EventConstants has, so I am extending from that class. This way, UserMediaConstants inherits all the properties of EventConstants. UserMediaConstants does not need anything special yet, that is why in itself, it has nothing defined. So yes, this is actually complete. (Its Object Oriented Programming)